components/portal/widgets/directives.js

```diff
@@ -48,8 +61,22 @@ define(['angular', 'require'], function(angular, require) {
       },
       templateUrl: require.toUrl('./partials/compact-widget-card.html'),
       controller: 'WidgetCardController',
+      link: {
+        post: function(scope, element, attrs) {
```

Comment:
:question: Could analytics capture be provided as a service that could be applied to any directive?